tools/iree-benchmark-module-main.cc

```diff
@@ -437,20 +437,20 @@ class IREEBenchmark {
 
     // Order matters. Tear down modules first to release resources.
     inputs_.reset();
-    iree_vm_context_release(context_);
-    iree_vm_module_release(main_module_);
-    iree_vm_instance_release(instance_);
+    context_.reset();
+    main_module_.reset();
+    instance_.reset();
 
     // Tear down device last in order to get accurate statistics.
-    if (device_allocator_ && FLAG_print_statistics) {
-      IREE_IGNORE_ERROR(
-          iree_hal_allocator_statistics_fprint(stderr, device_allocator_));
+    if (!!device_allocator_ && FLAG_print_statistics) {
+      IREE_IGNORE_ERROR(iree_hal_allocator_statistics_fprint(
+          stderr, device_allocator_.get()));
     }
```

Comment:
Is this `!!device_allocator_` intended? Why not just `device_allocator_`? Maybe find another way to express "ref has a value"?

Reply:
for some reason the bool operator stuff doesn't work - I'll see if I can figure it out

Reply:
ok, thanks. (LGTM either way, just this nit)

Reply:
fixed (if gcc is happy)! 🤞